packages/jw-player/src/lib/cxl-jwplayer-styles.js

```diff
@@ -0,0 +1,195 @@
+import { css } from 'lit';
+
+export const cxlJWPlayerStyles = css`
```

Comment:
@anoblet same here, maybe `scss` file should be used, and generate this css like `cxl-ui` does?